composer.json

```diff
@@ -13,11 +13,14 @@
         "php": "^7.2"
     },
     "conflict": {
-        "phpunit/phpunit": "<8.0"
+        "phpunit/phpunit": "<8.0",
+        "contao/core-bundle": "<4.9"
```

Comment:
This is not necessary. Contao 4.8 uses contao/test-case@3.* so contao/test-case@4.* is only used by Contao 4.9+ anyway.